### PR TITLE
Update tests sheet 

### LINKED ISSUE #21: Always include muon and chromium update to test plans
Always include muon and chromium update to test plans.

Currently they are not included unless they are tagged with `QA/test-plan-specified` (I usually add the label). They should always be included to the test runs.

### LINKED ISSUE #54: add Twitch test cases to Ledger Media
Add Test Cases for Twitch (maybe to existing Ledger Media section) for browser-laptop test plan. Maybe can use test cases from https://github.com/brave/browser-laptop/issues/13139#issuecomment-368931365 as a basis

### LINKED ISSUE #61: Add test for reopen closed window keyboard shortcut

### LINKED ISSUE #62: Merge wiki template and cr template

### PR DESCRIPTION
**Pull request by srirambv**
Merged Wikitemplate and Wikitemplate-cr-upgrade tests 
Fixes #62

added test for reopen closed window keyboard shortcut
 fixes #61 

Added YT/Twitch test cases 
fixes #54

Added muon/chrome checks 
fixes #21

### CODE REVIEW

File: braveautogen-chromium-upgrade.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 secret_file = open('github.secret', 'r')
 token_string = secret_file.readline().rstrip("\n\r")
 
-wiki_file = open('wikitemplate-cr-upgrade.md', 'r')
+wiki_file = open('wikitemplate.md', 'r')
```
Comment (kjozwiak) [inline braveautogen-chromium-upgrade.py]: we'll probably want to remove `braveautogen-chromium-upgrade.py` as it's basically going to be same thing as `braveautogen.py`.

File: wikitemplate.md [hunk 2]
```diff
@@ -168,9 +112,39 @@
...
+### Widevine
+  - [ ] Test that you can log into Netflix and start a show
+### Flash tests
+  - [ ] Test that flash placeholder appears on http://www.homestarrunner.com
+  - [ ] Test with flash enabled in preferences, auto play option is shown when visiting http://www.homestarrunner.com
+
+## Autofill tests
+
+- [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields
+- [ ] Verify clicking submit button on a form doesn't crash the browser
```
Comment (kjozwiak) [inline wikitemplate.md]: I'm not seeing a `Submit` button anywhere on the page. Just a `Reset` button which isn't crashing `0.22.667`. We might want to use one of the examples from known cases that the community has provided us with.

File: wikitemplate.md [hunk 3]
```diff
@@ -228,3 +220,18 @@ Do not forget to make a backup of your entire `~/Library/Application\ Support/Br
 
 - [ ] Test that updating using `BRAVE_UPDATE_VERSION=0.8.3` env variable works correctly.
 - [ ] Test that using `BRAVE_ENABLE_PREVIEW_UPDATES=TRUE` env variable works and prompts for preview build updates.
+
+# CR Upgrade specific tests
```
Comment (kjozwiak) [inline wikitemplate.md]: I don't think we need `CR specific tests`, we should probably be going through these cases every time we go through a pass. Performance wise, we should be making sure each release hasn't regressed in terms of performance. With the content test, it's easy enough to verify that we could probably add it into the general pass and not place it under `CR specific tests`. Thoughts @srirambv & @LaurenWags?

File: wikitemplate.md [hunk 4]
```diff
@@ -202,14 +176,32 @@
...
+- [ ] Verify wallet is auto created after enabling payments
+- [ ] Verify monthly budget and account balance shows correct BAT and USD value
+- [ ] Click on `add funds` and click on each currency and verify it shows wallet address and QR Code
+- [ ] Verify that Brave BAT wallet address can be copied
+- [ ] Verify adding funds via any of the currencies flows into BAT Wallet after specified amount of time
+- [ ] Verify adding funds to an existing wallet with amount, adjusts the BAT value appropriately
+- [ ] Change min visit and min time in advance setting and verify if the publisher list gets updated based on new setting
+- [ ] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table
+- [ ] Check that disabling payments and enabling them again does not lose state
+- [ ] Upgrade from older version
```
Comment (kjozwiak) [inline wikitemplate.md]: We should still test that upgrading from and older version to a newer version works in terms of not losing ledger data, but I don't think we need the cases where we check for overlays/bouncing balls etc. We can change this portion to basically test if an update keeps the ledger intact when updated from a version of Brave that's 2-3 versions back. Thoughts @srirambv & @LaurenWags?

File: wikitemplate.md [hunk 5]
```diff
@@ -168,9 +112,39 @@
 - [ ] Test zoom saved when you navigate within a single origin site
 - [ ] Test that navigating to a different origin resets the zoom
 
+## Printing
+
+- [ ] Test that you can print a PDF
+
+## Extensions/Plugins tests
+
+- [ ] Enable each extension one by one under `about:preferences#extensions` and ensure that the browser doesn't become unresponsive
```
Comment (kjozwiak) [inline wikitemplate.md]: nit: Space after this one

**Comment by srirambv** [comment]
Updated based on review comments. Fixes #64 too

File: wikitemplate.md [hunk 6]
```diff
@@ -201,15 +186,37 @@
...
+- [ ] Upgrade from older version
+  - [ ] Verify the wallet isn't corrupted upon upgrade (balance is retained and wallet backup code isn't corrupted)
+  - [ ] Verify publishers list is not lost after upgrade
+### Ledger Media (To be verified on YouTube and Twitch) 
+  - [ ] Visit any YouTube/Twitch video in a normal/session tab and ensure the video publisher name is listed in ledger table
+  - [ ] Visit any YouTube/Twitch video in a private tab and ensure the video publisher name is not listed in ledger table
+  - [ ] Visit any live YouTube/Twitch video and ensure the time spent is shown under ledger table
+  - [ ] Visit any embeded YouTube/Twitch video and ensure the video publisher name is listed in ledger table
+  - [ ] Ensure total time spent is correctly calculated for each publisher video
+  - [ ] Ensure total time spent is correctly calculated when switching to YouTube video from an embeded video
```
Comment (kjozwiak) [inline wikitemplate.md]: Should be ` when switching to YouTube/Twitch video` as we're testing embedded Twitch in the following case:

* `Visit any embeded YouTube/Twitch video and ensure the video publisher name is listed in ledger table`

File: wikitemplate.md [hunk 7]
```diff
@@ -201,15 +186,37 @@
...
+  - [ ] Ensure YouTube/Twitch publishers are not listed when `Allow contributions to video` is disabled in adavanced settings
+  - [ ] Ensure existing YouTube/Twitch publishers are not lost when `Allow contributions to video` is disabled in adavanced settings
+  - [ ] Ensure YouTube/Twitch publishers is listed but not included when `auto-include` is disabled
+  - [ ] Update Advanced settings to different time/visit value and ensure YouTube/Twitch videos are added to ledger table once criteria is met
+  - [ ] Perform a contribution while YouTube/Twitch channels are included on the ledger. Ensure the channels are listed on the contribution statement
+  - [ ] Verify able to delete YouTube/Twitch publishers from ledger table
```
Comment (kjozwiak) [inline wikitemplate.md]: nit: `Verify that you are able to delete a YouTube/Twitch publishers from the ledger table`

File: wikitemplate.md [hunk 8]
```diff
@@ -201,15 +186,37 @@
...
+  - [ ] Verify able to re-add YouTube/Twitch publishers to ledger table
```
Comment (kjozwiak) [inline wikitemplate.md]: nit: `Verify that you are able to re-add a YouTube/Twitch publishers to the ledger table`

**Pull request merged**